libs/evcs-service/src/main/java/uk/gov/di/ipv/core/library/service/EvcsService.java

```diff
@@ -24,6 +25,7 @@
 import static uk.gov.di.ipv.core.library.enums.EvcsVCState.PENDING_RETURN;
 
 public class EvcsService {
+    public static final EvcsVcProvenance DEFAULT_PROVENANCE_ONLINE = EvcsVcProvenance.ONLINE;
```

Comment:
Why do we need another variable here - can we just use `EvcsVcProvenance.ONLINE`?

Reply:
Done... got confused as its default at EVCS end, thought to mention it.